### DISCUSSION TITLE
[backend] Rescript & Typescript

### DISCUSSION THREAD

**Discussion by err0r500** [realworld-apps/realworld#936] [WIP Implementations]
Hi,

I just started a new implementation of the realworld app in [rescript](https://rescript-lang.org/) (for the domain modelling) & typescript (for adapters implementations). 

I try to reach the road blocks as fast as possible but it seems to be ok now (I spent quite some time trying to figure out how to make both languages work nicely together).

Here's the repo : https://github.com/err0r500/realworld-app-backend-rescript-ts 

Best, ;)

**Comment by geromegrignon** [comment]
Hi, awesome!
